tests/queries/0_stateless/03411_dynamic_resize_filesystem_cache_3.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+# In this test we only increase cache size two times and then decrease it back.
```

Comment:
It seems the cache size is increased once? – There is one sed search-replace with an increase, and another with a decrease.

Reply:
I meant two times is increasing the max size by 2 - `multiply(max_size, 2)`, will make the comment better